### PR TITLE
chore: adds custom scrollbar in table

### PR DESCRIPTION
**Pull request by riyalohia**
### What does this implement/fix? Explain your changes.

...

### Does this close any currently open issues?

...

### Any other comments?

...

### Dependent PRs/Commits

...

### Describe breaking changes, if any.

...

### Checklist

Check all those that are applicable and complete.

- [ ] Merged with latest `master` branch

### CODE REVIEW

**Review by satyamyadav — CHANGES_REQUESTED** [review]
@riyalohia  Please rebase your branch with current master branch.

File: css/src/components/grid.css [hunk 1]
```diff
@@ -38,6 +38,16 @@
     border-right: 3px solid var(--secondary);
 }
 
+.Grid::-webkit-scrollbar {
+    height: var(--spacing);
+    width: var(--spacing);
```
Comment (aditya-kumawat) [inline css/src/components/grid.css]: Update size to --spacing-l.

File: css/src/components/grid.css [hunk 2]
```diff
@@ -38,6 +38,16 @@
...
+}
+
+.Grid::-webkit-scrollbar-thumb {
+    background: var(--secondary);
+    border-radius: var(--spacing-m);
```
Comment (aditya-kumawat) [inline css/src/components/grid.css]: Update size to --spacing

**Comment by aditya-kumawat** [comment]
@riyalohia Please update PR/commit name as per convention.

**Review by satyamyadav — APPROVED** [review]

**Pull request merged**